### PR TITLE
chore(release): elements-core v7.1.0

### PR DESCRIPTION
**Pull request by mmiask**
Needed for: https://github.com/stoplightio/platform-internal/issues/6253

### CODE REVIEW

File: packages/elements-dev-portal/package.json [hunk 1]
```diff
@@ -47,7 +47,7 @@
   },
   "dependencies": {
     "@fortawesome/free-solid-svg-icons": "^5.10.2",
-    "@stoplight/elements-core": "^7.0.0",
+    "@stoplight/elements-core": "~7.0.0",
```
Comment (marcelltoth) [inline packages/elements-dev-portal/package.json:50]: Can we enforce >= 7.0.1 here? According to our best knowledge 7.0.0 does **not** work, so would you make this `~7.0.1` instead? Same for `elements`.

**Review by marcelltoth — CHANGES_REQUESTED** [review]

**Review by marcelltoth — APPROVED** [review]
You're right, it's not 7.0.1 but 7.1.0. Looks great now, thank you!

**Review by mpodlasin — COMMENTED** [review]
LGTM

**Comment by mpodlasin** [comment]
Lol, I thought it's still open for some reason :D

**Pull request merged**